### DISCUSSION TITLE
Soft CC VS Hard CC Defiance Break

### DISCUSSION THREAD

**Discussion by mightyteapot** [HardstuckGuild/GW2-Tooltips.js#60] [General]
Potential clarity/misleading issue that I can see happening. Right now we are combining soft cc and hard cc in the defiance bar totals, this is of course perfectly accurate however in any group setting (fractals, open world, raids etc) pretty much all conditions will be applied to the boss permanently which means that the defiance bar values we list are somewhat misleading because only the hard CC components will count in many situations.

Quite hard to solve this as I do think it is very useful to express how powerful soft CC can be.

Solutions could be the following:

1. Removing soft cc from the defiance bar totals, could work but might look a bit odd as the numbers not fully adding up may confuse users. Also would probably introduce some 'jank' because we'd want to include 'rare' conditions like immobilize + fear, but not cripple etc.
2. color coding soft and hard cc, display two numbers in the total defiance one for hard one for soft. This retains all information but could end up very wordy and bloated.

Bit of an interesting one; needs more thought.

**Comment by SaculRennorb** [comment]
So I'm much more in favor of the second idea, I don't like removing the information for such an arbitrary reason and having a mismatch in the numbers will more likely than not just repeatedly be reported as a bug. 

For now I wont bother implementing anything related to this, but we could look into it in the future.

This would also imply changes to the API as there currently is no distinction between those two types of defiance damage and it only returns either nothing or one numeric value per fact.
